firmware/raspi/boardinterface.py

```diff
@@ -73,16 +73,6 @@ def is_capture(self, uci_move):
         '''
         return self.chess_board.is_capture(self.chess_board.parse_uci(uci_move))
 
-    # @staticmethod
-    # def get_chess_coords_from_square(square):
-    #     '''Converts chess square to a BoardCell.
-
-    #     Example: a1 <=> [0, 0], h8 <=> [7, 7], regardless of whether human plays white or black
-    #     pieces.
-    #     '''
-    #     # Nums correspond to row (rank), letters correspond to col (files)
-    #     return util.BoardCell(ord(square[1]) - ord('1'), ord(square[0]) - ord('a'))
-
     def get_board_coords_from_square(self, square):
```

Comment:
Why did you decide get_chess_coords_from_square should be moved at the end of the uti.py file?

Reply:
I see now that I didn't need to. My thinking was that I didn't want to do `from boardinterface import Engine` in the `vision` file. Would you prefer I do that instead? I can change it back.

Reply:
No, I think that was a good change and was just wondering why.

Reply:
Sounds good!